app/dashboard/templates/bounty/details.html

```diff
@@ -444,9 +444,69 @@ <h5 class="bounty-heading">{% trans "Funder" %}</h5>
   <script src="{% static "v2/js/bootstrap.min.js" %}"></script>
   <script src="{% static "v2/js/clipboard.js" %}"></script>
   <script>
-    var bootstrapTooltip = $.fn.tooltip.noConflict()
-    var bootstrapModal = $.fn.modal.noConflict()
-    $.fn.bootstrapModal = bootstrapModal
+    let bootstrapTooltip = $.fn.tooltip.noConflict()
+    let bootstrapModal = $.fn.modal.noConflict()
+  </script>
+  <script>
+    $.fn.runTooltip = bootstrapTooltip
+
+    $('[data-toggle="popover"]').popover()
+
+    $('body').popover({
+      selector: '.bounty_row',
```

Comment:
can we change this selector for `.user-popover` to be more modular so we can use it in other places too?